.github/workflows/docker.yml

```diff
@@ -46,6 +53,20 @@ jobs:
             *.docker.com:443
             gcr.io:443
             repo.maven.apache.org:443
+            amazonlinux.default.amazonaws.com:443
+            amazonlinux.default.amazonaws.com:80
+            cdn.amazonlinux.com:443
+            rpm.nodesource.com:443
+            registry.npmjs.org:443
+            pypi.org:443
+            *.pypi.org:443
+            api.snapcraft.io:443
+            files.pythonhosted.org:443
+            acghubeus1.actions.githubusercontent.com:443
```

Comment:
This line seems fragile (the 1 at the end would, to me, indicate that there are other possiblilities?) but, if that works :shrug: